packages/caldav/README.md

```diff
@@ -0,0 +1,33 @@
+# timeld - CalDAV integration
+
+This packages supports live update of timesheet entries:
+- _from_ a CalDAV end-point
+- _to_ **timeld**.
+
+## installation & configuration
+
+This integration package is included in the gateway.
```

Comment:
I'd suggest capitalising "Gateway", to ensure readers know the documentation is referring to a specific timeld component.